java-examples/src/main/java/edu/practice/algos/sort/impl/MergeSorting.java

```diff
@@ -0,0 +1,57 @@
+package edu.practice.algos.sort.impl;
+
+public class MergeSorting {
```

Comment:
I dont understand the difference between MergeSort and MergeSorting... give distingushable names.

Reply:
abhi ismein koi changes nhi kiye bt i will revisit this merge sort again